### PR TITLE
6353: Get api provider from bcl

### PR DESCRIPTION
**Pull request by kat1906**
This will fail the checks until the apiProvider is merged into BCL

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
     "webpack-cli": "^3.3.10"
   },
   "jest": {
+    "testEnvironment": "node",
```
Comment (alibombaye) [inline package.json:85]: Just out of interest what does this do?

File: app/endpoints.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  postAddOrg: () => `${organisationApiUrl}/api/v1/Organisations`,
+
+  postAddUser: opts => `${identityServerUrl}/api/v1/Organisations/${opts.organisationId}/Users`,
+  postUserStatus: opts => `${identityServerUrl}/api/v1/Users/${opts.userId}/${opts.status}`,
+
+  // PUT endpoins
+  putUpdateOrganisation: opts => `${organisationApiUrl}/api/v1/Organisations/${opts.organisationId}`,
+};
+
+export const getEndpoint = ({ endpointLocator, options }) => endpoints[endpointLocator](options);
```
Comment (BenBJSS) [inline app/endpoints.js:24]: Should this go into the library?

File: package.json (hunk 1 above)
Comment (kat1906) [reply package.json:85]: was getting cors error before that

File: app/endpoints.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  getUsers: opts => `${identityServerUrl}/api/v1/Organisations/${opts.organisationId}/Users`,
+  getUserById: opts => `${identityServerUrl}/api/v1/Users/${opts.userId}`,
+
+  // POST endpoints
+  postAddOrg: () => `${organisationApiUrl}/api/v1/Organisations`,
+
+  postAddUser: opts => `${identityServerUrl}/api/v1/Organisations/${opts.organisationId}/Users`,
+  postUserStatus: opts => `${identityServerUrl}/api/v1/Users/${opts.userId}/${opts.status}`,
+
+  // PUT endpoins
```
Comment (BenBJSS) [inline app/endpoints.js:20]: Comment spelling triggers OCD

File: package.json (hunk 1 above)
Comment (kat1906) [reply package.json:85]: but it works without so can remove

File: app/endpoints.js (hunk 2 above)
Comment (kat1906) [reply app/endpoints.js:24]: that would mean sending all the endpoints into the librry?

File: app/endpoints.js (hunk 3 above)
Comment (kat1906) [reply app/endpoints.js:20]: :D good spot

File: app/endpoints.js (hunk 2 above)
Comment (BenBJSS) [reply app/endpoints.js:24]: I mean just the getEndpoint function

Comment (kat1906) [reply app/endpoints.js:24]: but we would need to pass all the endpoints into the getEndpoint function

Comment (BenBJSS) [reply app/endpoints.js:24]: You're right I'm being stupid here, was thinking it worked differently to how it does

**Pull request merged**